mappings/net/minecraft/entity/EntityContextImpl.mapping

```diff
@@ -8,4 +8,4 @@ CLASS net/minecraft/class_3727 net/minecraft/entity/EntityContextImpl
 	METHOD <init> (ZDLnet/minecraft/class_1792;)V
 		ARG 1 descending
 		ARG 2 minY
-		ARG 4 heldItem
+		ARG 3 heldItem
```

Comment:
Shouldn't this stay arg4 as double uses 2 slots

Reply:
Was this done using CFR or Procyon?